src/lib.ts

```diff
@@ -64,7 +64,7 @@ export class SecretKey {
     if (skBytes.length !== SECRET_KEY_LENGTH) {
       throw new ErrorBLST(BLST_ERROR.BLST_INVALID_SIZE);
     }
-    if (isZeroBytes(skBytes)) {
+    if (crypto.timingSafeEqual(skBytes, new Uint8Array(SECRET_KEY_LENGTH))) {
```

Comment:
Can you create `new Uint8Array(SECRET_KEY_LENGTH)` at the module scope to prevent having to allocate it on each check?